.github/workflows/lint-with-vale.yml

```diff
@@ -0,0 +1,19 @@
+---
+name: Linting with Vale on pull request
+on: [pull_request]
+
+jobs:
+  vale:
+    name: Linting with Vale
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: errata-ai/vale-action@reviewdog
+        with:
+          filter_mode: diff_context
+          vale_flags: "--no-exit --minAlertLevel=error --glob=!*/doc-*/common/*"
```

Comment:
*.adoc will prevent the sym links problem

Reply:
you mean `--glob=*.adoc`?

Reply:
Yes

Reply:
Doesn't work for me locally.

Running it with `--glob='*.adoc'` (quoted, so my shell doesn't interpret the *) and it spins for minutes now.

Reply:
but now I'm  not certain. I have not used that flag.